trusted_shuffler/server/src/http.rs

```diff
@@ -31,10 +31,7 @@ struct HttpRequestHandler {
 
 #[async_trait]
 impl RequestHandler for HttpRequestHandler {
-    async fn handle(
-        &self,
-        request: TrustedShufflerRequest,
-    ) -> anyhow::Result<TrustedShufflerResponse> {
+    async fn handle(&self, request: PlainRequest) -> anyhow::Result<PlainResponse> {
```

Comment:
Not sure about this terminology / naming. The correct word would be "plaintext", not "plain", so "PlainResponse" does not seem accurate to me. Also, that is usually contrasted by "encrypted" instead of "secret".

So personally I would suggest "PlaintextResponse" and "EncryptedResponse", and there would be "decrypt" / "encrypt" functions to convert between them.

WDYT?

Reply:
SG.